### PR TITLE
docs: 3.25.3 changelog

### PR DESCRIPTION
**Pull request by ycjcl868**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 📝 Changelog

- 🐞 Fix TimePicker disabled item style when focussed. [#19812](https://github.com/ant-design/ant-design/pull/19812) [@yoyo837](https://github.com/yoyo837)
- 🐞 Fix Menu.Item link style inside Badge. [#19810](https://github.com/ant-design/ant-design/pull/19810)
- 🐞 Fix Upload `picture-card` type, for crashed style when list item is removing. [#19783](https://github.com/ant-design/ant-design/pull/19783) [@qq645381995](https://github.com/qq645381995)
- 🇳🇱 Update `sk-SK` locales. [#19787](https://github.com/ant-design/ant-design/pull/19787) [@Kamahl19](https://github.com/Kamahl19)
- TypeScript
  - ⚡️ Export interface types in Tooltip. [19846](https://github.com/ant-design/ant-design/pull/19846) [@kachkaev](https://github.com/kachkaev)
### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/release-3.25.3/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/release-3.25.3/CHANGELOG.zh-CN.md)

### CODE REVIEW

**Review by shaodahong — APPROVED** [review]

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ timeline: true
 
 ---
 
+## 3.25.3
+
+`2019-11-24`
+
+- 🐞 修复 TimePicker 禁用项 focus 时的样式问题。 [#19812](https://github.com/ant-design/ant-design/pull/19812) [@yoyo837](https://github.com/yoyo837)
+- 🐞 修复 MenuItem 链接在 Badge 内的样式问题。[#19810](https://github.com/ant-design/ant-design/pull/19810)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: Menu.Item

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ timeline: true
 
 ---
 
+## 3.25.3
+
+`2019-11-24`
+
+- 🐞 Fix TimePicker disabled item focus style. [#19812](https://github.com/ant-design/ant-design/pull/19812) [@yoyo837](https://github.com/yoyo837)
+- 🐞 Fix Menu.Item link style inside Badge. [#19810](https://github.com/ant-design/ant-design/pull/19810)
+- 🐞 Fix Upload `picture-card` type, list item remove style. [#19783](https://github.com/ant-design/ant-design/pull/19783) [@qq645381995](https://github.com/qq645381995)
```
Comment (orzyyyy) [inline CHANGELOG.en-US.md]: Fix Upload `picture-card` type, for crashed style when list item is removing.

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ timeline: true
 
 ---
 
+## 3.25.3
+
+`2019-11-24`
+
+- 🐞 Fix TimePicker disabled item focus style. [#19812](https://github.com/ant-design/ant-design/pull/19812) [@yoyo837](https://github.com/yoyo837)
```
Comment (orzyyyy) [inline CHANGELOG.en-US.md]: Fix TimePicker disabled item style when focussed.

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,17 @@ timeline: true
 
 ---
 
+## 3.25.3
+
+`2019-11-24`
+
+- 🐞 修复 TimePicker 禁用项 focus 时的样式问题。 [#19812](https://github.com/ant-design/ant-design/pull/19812) [@yoyo837](https://github.com/yoyo837)
+- 🐞 修复 Menu.Item 链接在 Badge 内的样式问题。[#19810](https://github.com/ant-design/ant-design/pull/19810)
```
Comment (orzyyyy) [inline CHANGELOG.zh-CN.md]: 修复 Menu.Item 在 Badge 内时，a 标签始终处于 active 状态的问题。

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,17 @@ timeline: true
...
+- 🐞 修复 Upload 类型为 `picture-card` 时，列表删除的样式问题。[#19783](https://github.com/ant-design/ant-design/pull/19783) [@qq645381995](https://github.com/qq645381995)
```
Comment (orzyyyy) [inline CHANGELOG.zh-CN.md]:
```diff
- 列表删除的样式问题。
+ 列表删除时的样式问题。
```

File: CHANGELOG.zh-CN.md (hunk 4 above)
Comment (ycjcl868) [reply CHANGELOG.zh-CN.md]: Menu.Item 的链接在 Badge 内时

**Pull request merged**